dot/parachain/network-bridge/messages/pov_fetching.go

```diff
@@ -13,14 +13,19 @@ import (
 // PoVFetchingRequest represents a request to fetch the advertised collation at the relay-parent.
 type PoVFetchingRequest struct {
 	// Hash of the candidate for which we want to retrieve a Proof-of-Validity (PoV).
-	CandidateHash parachaintypes.CandidateHash
+	CandidateHash parachaintypes.CandidateHash `scale:"1"`
```

Comment:
So that you know, adding scale tags is not required as long as the order of fields is correct.
In this particular case, we have only one field, so there is no chance we get incorrect encoding because of the order of the fields.

In short, adding a scale tag doesn't make any difference here.

Reply:
The index will start at 0 if not specified.